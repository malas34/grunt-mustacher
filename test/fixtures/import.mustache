<html>
    <head>
        <title>{{title}}</title>
    </head>
    <body>
        <h3>Variables</h3>
        <p>{{content}}</p>
        <h3>Include</h3>
            {{$include 'partials'}}
    </body>
</html>
