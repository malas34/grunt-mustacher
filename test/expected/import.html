<html>
    <head>
        <title>Titre de la page</title>
    </head>
    <body>
        <h3>Variables</h3>
        <p>Contenu de la page</p>
        <h3>Include</h3>
        <p>This is a partials - Titre de la page</p>
        <p>Contenu de la page</p>
    </body>
</html>
